module/VuFind/src/VuFind/View/Helper/Root/RecordDataFormatter.php

```diff
@@ -131,20 +151,32 @@ protected function render($driver, $field, $data, $options)
 
     /**
      * Create formatted key/value data based on a record driver and field spec.
+     * The first argument can be a descendant of RecordDriver/AbstractBase.
```

Comment:
I think you can just say RecordDriver in this context, for consistency with the use statement above.